### PR TITLE
Merge without_OA into master

### PR DESCRIPTION
**Pull request by boblinchuan**
BAG framework branches were merged. Time to merge these.
Github says these can't be auto-merged. Need to discover why.

### CODE REVIEW

File: workspace_setup/bag_submodules.yaml
Comment (boblinchuan) [inline workspace_setup/bag_submodules.yaml:1]: These need to be bumped later

**Review by boblinchuan — COMMENTED** [review]
Noted changes:
- Bumped bag prims, new res_standard
- layer map - whitespace, additional (unused) layers
- Gen config - new res_standard, removing unused diodes, noting CDL vs SPECTRE netlists
- tech params - updating m5, m6 widths; layer map bump
- Change to SKILL default over OA (as per without_OA)
- Change to psfbin default over psfxl

Submodules need to be bumped later, but this branch itself should be mergable.

**Comment by boblinchuan** [comment]
Please approve #4 before this so that they can all be merged together.

**Review by ayan-biswas — APPROVED** [review]

**Review by boblinchuan — COMMENTED** [review]
LGTM

**Pull request merged**